ui/hooks/useEthFiatAmount.js

```diff
@@ -23,10 +23,13 @@ export function useEthFiatAmount(
   const conversionRate = useSelector(getConversionRate);
   const currentCurrency = useSelector(getCurrentCurrency);
   const userPrefersShownFiat = useSelector(getShouldShowFiat);
-  const showFiat = overrides.showFiat ?? userPrefersShownFiat;
+  const showFiat = Boolean(overrides.showFiat ?? userPrefersShownFiat);
```

Comment:
this boolean needs to be updated to take into account the possibility of conversionRate not being present.